r/NEWS.md

```diff
@@ -19,19 +19,54 @@
 
 # arrow 8.0.0.9000
```

Comment:
Shouldn't this be 9.0.0?
```suggestion
# arrow 9.0.0
```

Reply:
ok, I see this is supposed to be done on the `utils-prepare.sh` script as with the other versions.

Reply:
Correct, we don't make this change manually